packages/discovery-react-components/src/components/SearchFacets/SearchFacets.tsx

```diff
@@ -219,7 +219,7 @@ const SearchFacets: FC<SearchFacetsProps> = ({
     performSearch({ ...searchParameters, collectionIds: [], offset: 0, filter: '' }, false);
   };
 
-  if (isLoading) {
+  if (isLoading && !aggregations) {
```

Comment:
if we don't want this change i can remove it - was just somewhat jarring to see the aggregations disappearing after every search

Reply:
I think let's not include this change for now, as it seems to cause a flicker for the category facets as multiselect before they have types.

Reply:
i added this back in with the non-storing of initial aggregations fetch - let me know if this works better now?